ETWAnalyzer/Commands/DumpCommand.cs

```diff
@@ -386,6 +386,8 @@ class DumpCommand : ArgParser
         " ETWAnalyzer -dump Exception -filedir xx.json" + Environment.NewLine +
         "[green]Show all exceptions and their throw times by using a filter which matches all exceptions[/green]" + Environment.NewLine +
         " ETWAnalyzer -dump Exception -filedir xx.json -type *exception*" + Environment.NewLine +
+        "[green]Print dump exception for each process and executable version information (version, ...). [/green]" + Environment.NewLine +
```

Comment:
"Print exception" instead of  "Print dump exception". 
"executable version information" is redundant with the later "(version, ....)"
"executable information" should be sufficient.

Reply:
changed